tests/file_storage/models.py

```diff
@@ -72,6 +72,9 @@ def pathlib_upload_to(self, filename):
     default = models.FileField(
         storage=temp_storage, upload_to="tests", default="tests/default.txt"
     )
+    db_default = models.FileField(
+        storage=temp_storage, upload_to="tests", db_default="tests/db_default.txt"
+    )
```

Comment:
I was wondering if we should add a test for an empty db default.. From the docs I'm unclear about the semantics of `db_default=""` which I think it's a valid value for this param?

Reply:
7bcbf8d4adad720215058e0156da387e05ec744e I'm not sure the blank case makes sense, so I would be tempted to drop this commit. However, the behavior between `default=""` and `db_default=""` is the same so I think we're ok for this ticket/backport

Reply:
Agreed we can drop this commit for this fix, though I wonder if there is any follow up (no release blocker) necessary? Feels odd to allow for an empty name when saving a file.

Reply:
Yeah I think a system check so that you don't have blank default values might make sense, probably needs a deprecation warning, they should set null=True I think
Probably needs a forum discussion